tests/cypress/tests-e2e/mouseposition.cy.js

```diff
@@ -122,13 +122,11 @@ describe('Test mouse position and interactions', () => {
             cy.intercept(/^http[s]?:\/\/(sys-s\.\w+\.bgdi\.ch|s\.geo\.admin\.ch)\//, {
                 body: { shorturl: shortUrl, success: true },
             }).as('shortlink')
-            cy.intercept(`**/api/qrcode/generate**`, {
```

Comment:
I have removed the old qr code test since now the on from the menu is imported, should there be a new test added?

Reply:
Because the qrcode and the whole social sharing component is used twice it doesn't make much sense to test on both place. Check if the qrcode is tested in the share menu, if yes no need to add a test case here. If not we should consider adding a component test for the social media component, please create a jira ticket for this task.
In any case I would add simple assertions making sure that the social buttons are present and visible (no need to tests their action)